### DISCUSSION TITLE
Sass and CSS Modules, first render not showing.

### DISCUSSION THREAD

**Discussion by zigzactly** [vercel/next.js#13025] [Help]
NextJs version: 9.4.0
Hi there, i have a basic app running which uses:
- a global sass import in _app.js
- Css modules imports per component

I notice that on first render of the app, the components are rendered without the sass stylings, both global and local. Wait 1 to 2 seconds, and the sass styles start to apply on the browser. Styles props are happening on first render though. App continues to work as expected after that first render.

Have not dig deep yet, but i suspect its something simple i did wrong. Like maybe a config i missed out?

**Comment by rafinskipg** [comment]
I am pretty sure that's a bug. Maybe you can report it. I stopped using scss modules just because of that, and moved to styled jsx components

**Comment by dani97** [comment]
I made a basic app with components with css modules and faced the same issue, but the lag is very small. I think Css modules are not server rendered.
